objectivec/GPBMessage.h

```diff
@@ -93,6 +93,10 @@ CF_EXTERN_C_END
 /// @note In DEBUG builds, the parsed message is checked to be sure all required
 ///       fields were provided, and the parse will fail if some are missing.
 ///
+/// @note It is possible the returned errors are also originated in
+///       GPBCodedInputStream and not only in GPBMessage.
```

Comment:
Maybe add to the end of this something like "See GPBCodedInputSteam.h for the error domain and codes." ?